js/main.js

```diff
@@ -93,6 +97,46 @@ $form.addEventListener('submit', swappingViews);
 var $ul = document.querySelector('ul');
 $ul.addEventListener('click', swappingViews);
 
+function deleteModal(event) {
+  var $overLay = document.querySelector('#overlay');
+  $overLay.classList.remove('hidden');
+
+  var $modal = document.querySelector('#modal');
+  $modal.classList.remove('hidden');
+
+}
+var $deleteButton = document.querySelector('.delete-button');
+$deleteButton.addEventListener('click', deleteModal);
+
+function modalFunction(event) {
+  if (event.target.matches('.confirm-button')) {
+    var $allLi = document.querySelectorAll('.list-entry');
+    for (var allLiIndex = 0; allLiIndex < $allLi.length; allLiIndex++) {
+      var deleteLi = document.querySelector('img');
```

Comment:
How is this line working? How is getting the first `img` element in the page getting you to the `li` to be edited? 🤔